src/dbt_client/dbtCloudIntegration.ts

```diff
@@ -265,42 +289,42 @@ export class DBTCloudProjectIntegration
 
   async runModel(command: DBTCommand) {
     this.addCommandToQueue(
-      DBTCloudProjectIntegration.QUEUE_OTHERS,
+      DBTCloudProjectIntegration.QUEUE_ALL,
```

Comment:
It will be useful for us to know why this is changed. Can you add some details?

Reply:
There is only one queue actually, so that's why I changed the name. This was part of the the various things I tried for dbt cloud to work around the session occupied error.